### ISSUE TITLE
Can I offer the dream of ACNL?

### ISSUE THREAD

**Issue by SODA4727** [PretendoNetwork/archival-tools#3]
Hello! I wanted to dump all ACNL dreams, so I'm currently dumping dreams of Nintendo Network using this tool. If it can help, can I offer this dump to you (Pretendo)?

**Comment by jonbarrow** [comment]
Thank you for the offer! However one of our team members, @InternalLoss, has created a backup for us already and is periodically running it to get any new updates

**Comment by SODA4727** [comment]
Okay! If you ever need a dump, please contact me! I'm going to dump a few more times!
+Can I convert this dump and actually apply it to the game? (Save file or Dream(Pretendo))

**Comment by jonbarrow** [comment]
No, the dumps created by this archiver cannot be used directly in game. They need to be processed first and then served through a server
